pkg/scalers/kafka_scaler.go

```diff
@@ -208,7 +208,7 @@ func (s *kafkaScaler) GetMetricSpecForScaling() []v2beta1.MetricSpec {
 }
 
 //GetMetrics returns value for a supported metric and an error if there is a problem getting the metric
-func (s *kafkaScaler) GetMetrics(ctx context.Context, merticName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error) {
+func (s *kafkaScaler) GetMetrics(ctx context.Context, metricName string, metricSelector labels.Selector) ([]external_metrics.ExternalMetricValue, error) {
```

Comment:
Line 236 uses this variable so you'd need to fix there too

Reply:
sorry, i have seen it during tgik and quickly try to fix it !

Reply:
No worries! Thanks for the help